src/orders/components/OrderCustomerAddressesEditDialog/utils.ts

```diff
@@ -0,0 +1,52 @@
+import {
+  CustomerAddresses_user_addresses,
+  CustomerAddresses_user_defaultShippingAddress
+} from "@saleor/customers/types/CustomerAddresses";
+
+import { getById } from "../OrderReturnPage/utils";
+
+export const flatten = (obj: unknown) => {
```

Comment:
I would move it to the main `misc`

Reply:
@Droniu can we not accomplish this using lodash?

Reply:
No, lodash `flatten` function works on arrays, not objects. IMO it's better to avoid lodash if something can be done in plain javascript without much hustle. However, I'm working on this function to use `.reduce()` instead.